proposals/resolution_merge.json

```diff
@@ -0,0 +1,73 @@
+{
+    "id": "resolution_merge",
+    "summary": "Resolution merging / pan-sharpening",
+    "description": "Resolution merging algorithms try to improve the spatial resolution of lower resolution bands (e.g. Sentinel-2 20M) based on higher resolution bands (e.g. Sentinel-2 10M).",
+    "categories": [
+        "cubes",
+        "optical"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "Data cube containing multiple spectral bands, with different source resolutions.",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "name": "high_resolution_bands",
+            "description": "A list of band names to use as 'high-resolution' band. These bands will remain unmodified.\n\nEither the unique band name (metadata field `name` in bands) or one of the common band names (metadata field `common_name` in bands). If unique band name and common name conflict, the unique band name has higher priority.\n\nThe order of the specified array defines the order of the bands in the data cube. If multiple bands match a common name, all matched bands are included in the original order.",
+            "schema": {
+                "type": "array",
+                "items": {
+                    "type": "string",
+                    "subtype": "band-name"
+                }
+            }
+        },
+        {
+            "name": "low_resolution_bands",
+            "description": "A list of band names for which the spatial resolution should be increased. These bands will be modified by the process.\n\nEither the unique band name (metadata field `name` in bands) or one of the common band names (metadata field `common_name` in bands). If unique band name and common name conflict, the unique band name has higher priority.\n\nThe order of the specified array defines the order of the bands in the data cube. If multiple bands match a common name, all matched bands are included in the original order.",
+            "schema": {
+                "type": "array",
+                "items": {
+                    "type": "string",
+                    "subtype": "band-name"
+                }
+            }
+        },
+        {
+            "name": "method",
```

Comment:
@przell  and I have removed the method parameter in #210. I guess we should align here or there.